packages/babel-generator/src/buffer.js

```diff
@@ -14,6 +14,7 @@ export default class Buffer {
     this._indent = format.indent.base;
     this.format = format;
     this.buf = "";
+    this.last = "";
```

Comment:
@gzzhanghao That's very interesting information. May I ask where/how did you learn that? I'd like to learn things like that as well :)